services/datamanager/builtin/capture_test.go

```diff
@@ -132,6 +119,8 @@ func TestDataCaptureEnabled(t *testing.T) {
 			// Set up capture directories.
 			initCaptureDir := t.TempDir()
 			updatedCaptureDir := t.TempDir()
+			mockClock := clk.NewMock()
+			clock = mockClock
```

Comment:
Took me a while to realize that this is being used by the data manager since I didn't see `clock` referenced anywhere else in this function. Can we add a comment?